python/ray/data/_internal/execution/interfaces/execution_options.py

```diff
@@ -89,6 +89,9 @@ class ExecutionOptions:
     Attributes:
         resource_limits: Set a soft limit on the resource usage during execution.
             This is not supported in bulk execution mode. Autodetected by default.
+        reserved_resources: Amount of reserved resources for non-Ray-Data
+            workloads. Ray Data will exlcude these resources when scheduling tasks,
+            unless resource_limits is manually set.
```

Comment:
```suggestion
        exclude_resources: Amount of reserved resources for non-Ray-Data
            workloads. Ray Data will exlcude these resources when scheduling tasks,
            unless resource_limits is manually set.
```

I just realized reserved_resources sounds a bit like resources reserved for Data, which is the opposite of what we want.